chipy_org/apps/meetings/forms.py

```diff
@@ -76,9 +76,26 @@ def __init__(self, request, *args, **kwargs):
         if self.instance.pk:
             del self.fields["email"]
 
+        meeting = kwargs.get("initial").get("meeting", None)
```

Comment:
This access pattern surprises me. Can you walk through why this is needed now and why we can't rely on `super()` to get some of this data?